src/Leaflet.Renderer.SVG.Tile.js

```diff
@@ -39,6 +39,16 @@ L.SVG.Tile = L.SVG.extend({
 		}
 	},
 
+	removeFrom: function (map) {
+		if (this.options.interactive) {
+			for (var i in this._layers) {
+				var layer = this._layers[i];
+				delete this._map._targets[L.stamp(layer._path)];
```

Comment:
There is indeed, it's `Layer.add/removeInteractiveTarget(domElement)`. The problem here is that the `FeatureLayer` subclasses are never properly added to the map (using `Layer.addTo()` or `Map.addLayer()`). As a result, the layer's `_map` member is never set which is used by the above methods.

One way to solve this would be to add calls to the FeatureLayer's `addTo()` methods. I don't see any obvious problems with this, except that it does a lot of work such as registering event handlers and firing events which isn't needed in this case since we know the FeatureLayers never escape to clients.

My preferred solution to this is to add a `Map.addInteractiveTarget(domElement, Evented)` call. Then `FeatureLayer` wouldn't need to subclass `Layer` anymore, which seems the wrong level of abstraction anyway, seeing as they don't exactly represent independent entities to be added or removed from maps and one is created for every feature from a vector tile.

My plan here was to just use the 'direct' route which is symmetric with the way `addTo()` is implemented. In the future, it would be nice to do the refactoring I describe above, I'm curious to hear your thoughts.

If you think it makes sense to initialise the `_map` member of the FeatureLayers properly and use Layer's `addInteractiveTarget()`, I can look into this as well.